src/entryPoints/AuthModule/UserInfo/UserInfoTypes.res

```diff
@@ -1,3 +1,4 @@
+type entity = [#Internal | #Organization | #Merchant | #Profile]
```

Comment:
Can we make this entity generic for whole dashboard so that team management module can also use this

Reply:
what you mean by generic